Cargo.toml

```diff
@@ -222,3 +222,4 @@ zksync_crypto_primitives = { path = "core/lib/crypto_primitives" }
 zksync_node_framework = { path = "core/node/node_framework" }
 zksync_eth_watch = { path = "core/node/eth_watch" }
 zksync_shared_metrics = { path = "core/node/shared_metrics" }
+zksync_commitment_generator = { path = "core/node/commitment_generator" }
```

Comment:
Not specific to this crate: Shouldn't this crate and other crates the `core/node` directory be a part of the workspace (i.e., be mentioned in `workspace.members` list)?

Reply:
A separate workspace doesn't really make sense for now because of a two-way dependency with the `core` / main workspace. It [turns out](https://doc.rust-lang.org/stable/cargo/reference/workspaces.html#the-members-and-exclude-fields) that all crates in the `node` dir are already members of the main workspace:

> All [path dependencies](https://doc.rust-lang.org/stable/cargo/reference/specifying-dependencies.html#specifying-path-dependencies) residing in the workspace directory automatically become members. Additional members can be listed with the members key, which should be an array of strings containing directories with Cargo.toml files.

(Which is expected: how would `*.workspace = true` keys work in the crate manifests otherwise?) I'd suggest to add crates to the main workspace explicitly so that it's more clear and there's no discrepancy with the `core/node/node_framework` crate, which *is* included explicitly into `workspace.members`.

Reply:
ah..yes..
did not know that 
> I'd suggest to add crates to the main workspace explicitly 

make sense